lib/ansible/modules/storage/netapp/na_ontap_interface.py

```diff
@@ -242,6 +249,8 @@ def set_options(self, options, parameters):
             options['is-auto-revert'] = 'true' if parameters['is_auto_revert'] is True else 'false'
         if parameters.get('admin_status') is not None:
             options['administrative-status'] = parameters['admin_status']
+        if parameters.get('force_subnet_association') is not None:
+            options['force-subnet-association'] = 'true' if parameters['force_subnet_association'] is True else 'false'
```

Comment:
the "is True" is not needed here:
options['force-subnet-association'] = 'true' if parameters['force_subnet_association'] else 'false'

Reply:
fixed! ;)

Reply:
You may need to push it again (look like everything failed on shippable for some reason)

Reply:
rebased after the recent push of #52283